src/MediaManagerBundle/Bridge/Chameleon/BackendModule/MediaManagerBackendModule.php

```diff
@@ -814,12 +814,14 @@ protected function moveMediaTreeNode()
     {
         $mediaTreeNodeId = $this->inputFilterUtil->getFilteredPostInput('id');
         $mediaTreeNodeParentId = $this->inputFilterUtil->getFilteredPostInput('parentId');
-        $position = (int) $this->inputFilterUtil->getFilteredPostInput('position');
+        $position = $this->inputFilterUtil->getFilteredPostInput('position');
 
         if (null === $mediaTreeNodeId || null === $mediaTreeNodeParentId || null === $position) {
             $this->returnGeneralErrorMessageForAjax();
         }
 
+        $position = (int) $position;
```

Comment:
In order to preserve the `null` check, I moved casting of `$position` to after  the check